pkg/rpm/horizon-cli/Makefile

```diff
@@ -71,7 +71,12 @@ rpmbuild: require-version
 	rm -rf $(RPMROOT)/BUILD/horizon-cli-*
 	rm -f $(RPMROOT)/SRPMS/$(RPMNAME)*rpm $(RPMROOT)/RPMS/$(rpm_arch)/$(RPMNAME)*rpm $(RPMROOT)/RPMS/$(rpm_arch)/$(RPMNAME)*rpm.gz
 	rpmbuild --target $(rpm_arch)-linux $(RPM_OPTS) -ba $(RPMNAME).spec
-	gzip --keep $(RPMROOT)/RPMS/$(rpm_arch)/$(RPMNAME)-$(VERSION)-$(BUILD_NUMBER).$(rpm_arch).rpm
+	$(eval target_rpm_path := $(RPMROOT)/RPMS/$(rpm_arch))
+	$(eval target_rpm_name := $(RPMNAME)-$(VERSION)-$(BUILD_NUMBER).$(rpm_arch).rpm)
+	@# To save compatibility with previous versions of gzip (older than 1.6), it's
+	@#  recommended to use simple shell redirects for keeping original file (.rpm)
+	@# See https://unix.stackexchange.com/a/58814
+	gzip < $(target_rpm_path)/$(target_rpm_name) > $(target_rpm_path)/$(target_rpm_name).gz
```

Comment:
- Can't you have just a single variable that is the full path?
- Also, in the comment mention that it is the `--keep` flag that isn't available in older than 1.6, and that, for example, RHEL 7.x has older than 1.6 gzip

Reply:
Done